src/partials/catalog.html

```diff
@@ -0,0 +1,100 @@
+<section class="catalog-section">
+  <div class="container catalog-container">
+    <h2 class="catalog-name">Catalog</h2>
+    <ul class="catalog-list">
+      <li class="catalog-item">
+        <img
+          class="catalog-img"
+          srcset="
+            ./img/catalog/catalog1.png    1x,
+            ./img/catalog/catalog1-2x.png 2x
+          "
+          src="./img/catalog/catalog1.png"
+          alt="Fenix 7X Pro Sapphire Solar"
+        />
+        <div class="catalog-imfo">
+          <h3 class="catalog-item-name">Garmin Fenix 7X Pro Sapphire Solar</h3>
+          <p class="catalog-item-prc">€500</p>
+        </div>
+      </li>
+      <li class="catalog-item">
+        <img
+          class="catalog-img"
+          srcset="
+            ./img/catalog/catalog2.png    1x,
+            ./img/catalog/catalog2-2x.png 2x
+          "
+          src="./img/catalog/catalog2.png"
+          alt="Marq Athlete"
+        />
+        <div class="catalog-imfo">
+          <h3 class="catalog-item-name">Garmin Marq Athlete</h3>
+          <p class="catalog-item-prc">€450</p>
+        </div>
+      </li>
+      <li class="catalog-item">
+        <img
+          class="catalog-img"
+          srcset="
+            ./img/catalog/catalog3.png    1x,
+            ./img/catalog/catalog3-2x.png 2x
+          "
+          src="./img/catalog/catalog3.png"
+          alt="Descent MK1"
+        />
+        <div class="catalog-imfo">
+          <h3 class="catalog-item-name">Garmin Descent MK1</h3>
+          <p class="catalog-item-prc">€680</p>
+        </div>
+      </li>
+      <li class="catalog-item">
+        <img
+          class="catalog-img"
+          srcset="
+            ./img/catalog/catalog4.png    1x,
+            ./img/catalog/catalog4-2x.png 2x
+          "
+          src="./img/catalog/catalog4.png"
+          alt="D2 Delta PX"
+        />
+        <div class="catalog-imfo">
+          <h3 class="catalog-item-name">Garmin D2 Delta PX</h3>
+          <p class="catalog-item-prc">€380</p>
+        </div>
+      </li>
+      <li class="catalog-item">
+        <img
+          class="catalog-img"
+          srcset="
+            ./img/catalog/catalog5.png    1x,
+            ./img/catalog/catalog5-2x.png 2x
+          "
+          src="./img/catalog/catalog5.png"
+          alt="Fenix 6 Pro"
+        />
+        <div class="catalog-imfo">
+          <h3 class="catalog-item-name">Garmin Fenix 6 Pro Solar</h3>
+          <p class="catalog-item-prc">€400</p>
+        </div>
+      </li>
+      <li class="catalog-item">
+        <img
+          class="catalog-img"
+          srcset="
```

Comment:
It seems to me that srcset  should be for different screen widths and pictures should also be for all sizes